addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -138,6 +143,16 @@ var FormViewDialog = ViewDialog.extend({
                         },
                     });
                 }
+
+                if (!options.disable_multiple_selection && this.activeActions.delete && !options.isMobile) {
```

Comment:
Isn't this task about adding a way to delete x2m records in mobile view ? Why is this code targeting a use-case that is specifically not mobile then ?

Reply:
According to spec, we needed a generic system for deleting a x2m record. Previously to delete such record, we had to explicitly add 'x' icon on o2m / m2m kanban view (which won't be there in kanban cards like quotation line etc in mobile view). 
Now, there won't be 'x' icon anywhere. Instead, when opening x2m record : 
* In desktop it will open a regular modal, with newly added 'Remove' button at the right side of a footer
* In mobile it will open a Full Screen Modal, with trash icon on the top-right corner inside header.

Reply:
Ok